src/content/docs/usage/workspaces.mdx

```diff
@@ -64,42 +83,69 @@ daytona create --code
+
+   ```text
+      WORKSPACE | ✓ Request submitted
+      WORKSPACE | ✓ Creating workspace daytona
+      daytona   | Creating project daytona
+      daytona   | Pulling image...
+      daytona   | Pulling from daytonaio/workspace-project
+      ...
+      daytona   | Project daytona created
+      daytona   | Starting project daytona
+      daytona   | Project daytona started
+   ```
+
+2. Wait while Daytona sets up your Workspace. It handles all the initialization and configuration of your environment.
 
 <Aside type="tip">
 You can set the Project Builder at Workspace creation time using command-line flags.
 These flags only apply when creating Workspaces with a single Project.
--   To use the Custom Image Builder, set both `--custom-image` and `--custom-image-user` flags.
--   To use the Dev Container Builder, set the`--devcontainer-path` flag to the Dev Container configuration path within the repository.
--   To explicitly specify the Builder, set the`--builder` flag.
+- To use the Custom Image Builder, set both `--custom-image` and `--custom-image-user` flags.
+- To use the Dev Container Builder, set the`--devcontainer-path` flag to the Dev Container configuration path within the repository.
+- To explicitly specify the Builder, set the`--builder` flag.
     Accepted values are `auto`, `devcontainer`, or `none`.
 
-__Example__
+**Example**
+
 ```shell
 daytona create <REPO_URL> --custom-image=daytona-workspace:latest --custom-image-user=daytona
 daytona create <REPO_URL> --devcontainer-path=.devcontainer/devcontainer.json
 daytona create <REPO_URL> --builder=auto
 ```
+
 </Aside>
 
 ## List Workspaces
 
-You can list all previously created Workspaces.
+Daytona allows you to keep track of your development environments by listing all previously created Workspaces. This helps you manage multiple projects and ensure that you can easily access and maintain your Workspaces.
 
 1. Execute the following command to see a list of Workspaces:
 
    ```shell
    daytona list
    ```
 
+   Upon running this command, Daytona will display a list of your Workspaces with their details. You will be able to see the Workspace name, the repository it is connected to, the target of the Workspace, and its status.
+
+   ```text
+      Workspace        Repository          Target     Status
+      ─────────────────────────────────────────────────────────────────────
+      MyWorkspace      username/MyProject  local      RUNNING (30 minutes)
+   ```
+
 ## Open An Existing Workspace
 
-You can open an existing Workspace in your IDE.
+Daytona allows you to open an existing Workspace in your IDE. This enables you to quickly access and work on your projects in your preferred development environment. With Daytona, you can effortlessly switch between different Workspaces and maintain an efficient workflow.
 
 **Prerequisites**
```

Comment:
@stefanicjuraj are all these Prerequisites maybe overkill? perhaps just once on at the top "**A working installation of Daytona.**

  Verify your Daytona installation is up-to-date to avoid compatibility issues."